discord/components.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+
+class Component:
```

Comment:
This file doesn't fit in the planned file setup. Please look at #7 for how we are going to structure the models